pandas/util/_validators.py

```diff
@@ -269,6 +270,37 @@ def validate_bool_kwarg(
     return value
 
 
+def validate_na_arg(
+    value, name: str, allow_no_default: bool = False, allow_bool: bool = False
+):
+    """
+    Validate na arguments.
+
+    Parameters
+    ----------
+    value : object
+        Value to validate.
+    name : str
+        Name of the argument, used to raise an informative error message.
+    allow_no_default : bool, default False
+        Whether to allow ``value`` to be ``lib.no_default``.
+    allow_bool : bool, default False
+        Whether to allow ``value`` to be an instance of bool.
+
+    Raises
+    ______
+    ValueError
+        When ``value`` is determined to be invalid.
+    """
+    if allow_no_default and value is lib.no_default:
+        return
+    if allow_bool and isinstance(value, bool):
+        return
+    if isna(value):
+        return
+    raise ValueError(f"{name} must be a valid NA value; got {value}")
```

Comment:
```suggestion
    raise ValueError(f"{name} must be a bool (True/False) or a valid NA value; got {value}")
```

Reply:
BTW, I don't know if there is a case for which you are foreseeing `allow_no_default` and `allow_bool` to be used, but if they would actually be used, then the error message would also have to be updated to follow that.

(so for now could also simplify things leaving out the keywords?)

Reply:
Yes, looks good!